lhotse/features/fbank.py

```diff
@@ -54,6 +54,10 @@ def _feature_fn(self, *args, **kwargs):
     def feature_dim(self, sampling_rate: int) -> int:
         return self.config.num_mel_bins
 
+    @property
+    def padding_value(self) -> float:
+        return -1000.0
```

Comment:
Is there a reason to choose -1000.0 instead of `LOG_EPSILON` or just `0`?

Reply:
Check the discussion in the main thread.